controllers/fenceagentsremediation_controller.go

```diff
@@ -287,6 +287,16 @@ func (r *FenceAgentsRemediationReconciler) updateStatus(ctx context.Context, far
 	return nil
 }
 
+func getNodeName(far *v1alpha1.FenceAgentsRemediation) string {
```

Comment:
Can you add a function declaration that will include we take the node name from the CR name or the annotation?